### DISCUSSION TITLE
Link to extension docs

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#16850] [Contribute to Supabase]
We have 
https://supabase.com/docs/guides/database/extensions

which documents most/all of the extensions available on the dashboard. We should link from the dashboard to each one. 

https://supabase.com/dashboard/project/_/database/extensions

<img width="909" alt="CleanShot 2023-06-28 at 11 43 05@2x" src="https://github.com/supabase/supabase/assets/105593/de95a594-eb6f-4d86-a6ff-375eb5c6db82">

**Comment by GeekyADAMS** [comment]
Wanna take this up

**Comment by Yogirajpp** [comment]
What is the exact issue you want to solve?
